src/jbi/models.py

```diff
@@ -24,13 +24,18 @@ class Action(YamlModel):
     enabled: bool = False
     allow_private: bool = False
     parameters: dict = {}
+    _caller: Callable = PrivateAttr(default=None)
 
-    @functools.cached_property
-    def callable(self) -> Callable:
-        """Return the initialized callable for this action."""
+    def _initialize_caller(self):
         action_module: ModuleType = importlib.import_module(self.module)
-        initialized: Callable = action_module.init(**self.parameters)  # type: ignore
-        return initialized
+        initialized_caller: Callable = action_module.init(**self.parameters)  # type: ignore
+        self._caller = initialized_caller
+
+    def call(self, payload):
```

Comment:
Can't this be a property instead?

It confused my monday morning brain. 
We have two notions of actions. This one here, representing configuration entries, which would now have a `call()` method, and the whiteboard actions, which are callables, and would be called from here via the runner.

I think that if we could, it would be slightly easier to follow if the action model would only have attributes, and the runner be in charge to execute them.
If we can't, let's polish naming a bit maybe to see if it helps (we have `call()`, `__call__()`, `execute_action()`, ...)

Reply:
Good thinking @leplatrem. Addressed this in d78c61f.

Also, luckily, turns out a `@property` isn't normally included in the serialized object. So it was really just a matter of "caching" the initialized caller in the private field (instead of using `keep_untouched` and `functools.cached_property`) to have the callable be a property.

Also, I renamed `callable` -> `caller` just because I didn't want to mirror Python's built-in `callable` function.